core/chains/evm/gas/docs/FEE_HISTORY_ESTIMATOR.md

```diff
@@ -0,0 +1,56 @@
+# Fee History Estimator
+
+## Overview
+`Fee History` estimator is an EVM-based gas estimator that utilizes RPC calls to make gas price estimations. The estimator heavily relies on two RPC calls: `eth_gasPrice` and `eth_feeHistory`. It is built as a service and caches the calculated results in order to minimize overhead. It can bump past estimations by aggregating the result with the latest market prices. `Fee History` estimator supports both Legacy and Dynamic(EIP-1559) transactions. It can also handle chains that don't have a mempool and process transactions in an FCSF manner.
```

Comment:
```suggestion
`Fee History` estimator is an EVM-based gas estimator that utilizes RPC calls to make gas price estimations. The estimator heavily relies on two RPC calls: `eth_gasPrice` and `eth_feeHistory`. It is built as a service and caches the calculated results in order to minimize overhead. It can bump past estimations by aggregating the result from most recent blocks. `Fee History` estimator supports both Legacy and Dynamic(EIP-1559) transactions. It can also handle chains that don't have a mempool and process transactions in an FCSF manner.
```

Reply:
I didn't want to get into too much detail here, since we explain FHE's behavior below and the code has plenty of comments. Is there something in particular that doesn't make sense?

Reply:
Maybe in that case we can omit this detail here and let people learn this below? This line is trying to say that if the market has spiked above the bumped value, the estimator would use the relevant market price instead right?

Reply:
I wanted to quickly mention that bumping also takes into account market prices but not go into too much detail because then I would have to mention the caps and the limits. If this seems confusing, I can remove it instead.